Zend/zend_compile.c

```diff
@@ -3903,6 +4040,7 @@ int zend_compile_func_array_key_exists(znode *result, zend_ast_list *args) /* {{
 	}
 
 	zend_compile_expr(&needle, args->child[0]);
+	CG(short_circuiting) = 0;
```

Comment:
Why is this needed? One is one arg short circuiting and the other isn't?

Reply:
Hmm... Maybe this isn't even necessary. I'll check it out.

Reply:
Yep, wasn't. Removed it.